packages/beacon-node/src/chain/bls/multithread/index.ts

```diff
@@ -134,9 +136,14 @@ export class BlsMultiThreadWorkerPool implements IBlsVerifier {
     this.logger = logger;
     this.metrics = metrics;
     this.blsVerifyAllMultiThread = options.blsVerifyAllMultiThread ?? false;
+    this.blsAddVerificationRandomness = options.blsAddVerificationRandomness ?? true;
 
     // TODO: Allow to customize implementation
     const implementation = bls.implementation;
+    if (implementation === "herumi") {
+      // mult not implemented by base herumi library
```

Comment:
Worth creating an issue in `bls` repo?

Reply:
Yes very much so!! I mentioned this in passing on one of the calls we did together as a good idea for something that should be done.  An issue will def formalize it.  I did some research and found that there is a `mult` free function included in the `bls-eth-wasm` library but it was not so easily accessible.  It looks like its being exported but was not part of the TS definition so need to dig a bit to figure out how to get a hold of it.  There are a couple of factory functions that initialize the herumi library and it was not clear, without more research, on how to implement.  Should be possible though and potentially when you updated the library the new definitions expose the helper free functions!!!  TBD though.